web/src/components/ErrorMessage.tsx

```diff
@@ -1,23 +1,29 @@
 import React from 'react'
-import makeStyles from '@mui/styles/makeStyles'
+import { styled } from '@mui/material/styles'
 import { theme } from 'app/theme'
 
-interface Props {
-  error: string
-  message?: string
-  context?: string
-  marginTop?: number
-  marginBottom?: number
+const PREFIX = 'ErrorMessage'
+
+const classes = {
+  root: `${PREFIX}-root`
 }
 
-const useStyles = makeStyles(() => ({
-  root: {
+const Root = styled('div')(() => ({
+  [`&.${classes.root}`]: {
```

Comment:
Does whitespace matter? Should this be ```[`& .${classes.root}`]```

Reply:
No idea, I'll make that change. FYI there's a "Add a suggestion" button on GH code reviews for small code suggestions, `cmd+g`